### DISCUSSION TITLE
Nextcloud Integration

### DISCUSSION THREAD

**Discussion by Elliot-Construct** [Budibase/budibase#4854] [Ideas]
Hey there!

I am currently in the process of integrating my Nextcloud Instance with my Budibase instance to create more applications within the nextcloud portal.

I would love to see the possibilities of file-related integrations and am currently working on some Nextcloud Talk notification chatbots.

Cheers
Elliot!

**Comment by mjashanks** [comment]
Thanks Elliot!

Yea, this would be nice. One of the issues we have at the moment is that Budibase doesn't handle files all that well. 

For example, I'm sure you could connect to Nextcloud with it's rest API, and browse files etc. However, uploading and downloading files I suspect would be an issue. Definitely something we'd like to be better at.

**Comment by satonotdead** [comment]
Hey guys, that sounds great!

Do you know if a workaround is possible using API requests and/or webhooks? We want to use data from Talk on Nextcloud (that has integrated a good use of OpenAI).

**Reply by JCV67** [reply]
Some kind of a "fake" attachment collumn type would be helpful. The ability of draging a file or folder to a form field but, only the file name and its hyperlink to a windows or Linux file server, would be stored on the database.
